api/src/entities/user/user.model.ts

```diff
@@ -0,0 +1,37 @@
+import { Sequelize, Model, DataTypes } from 'sequelize';
+
+const sequelize = new Sequelize('postgres://postgres:123@localhost:5432/mentoring');
```

Comment:
I would definitely move this to .env in future PRs. Hardcoding login and password is strongly discouraged. It's easy to do with dotenv package or some alternatives.